docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.7"
+
+services:
+  subhub:
+    container_name: subhub
+    build:
+      context: .
+      args:
+        AWS_ACCESS_KEY_ID: "AKIAIOSFODNN7EXAMPLE"
+        AWS_SECRET_ACCESS_KEY: "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
+        STRIPE_API_KEY: "sk_test_123"
+        SUPPORT_API_KEY: "support_test"
+        # NOTE: This enables debug logging of the Stripe API when running in docker-compose.
+        STRIPE_LOG: "debug"
+        LOCAL_FLASK_PORT: 5000
+    ports:
+      - "5000:5000"
```

Comment:
The environment variables could also go into an docker-compose environment file.  Just say the word and I happily move them.